ci/azure/linux.yml

```diff
@@ -12,30 +12,63 @@ jobs:
     AZURECI: 1
     COMPOSE_FILE: ci/docker-compose.yml
     PYTHONHASHSEED: "random"
-    PYTEST_MARK_EXPRESSION: "not udf"
-    BACKENDS: "clickhouse impala kudu-master kudu-tserver mysql omniscidb parquet postgres sqlite"
+    # TODO: omniscidb should be in BACKENDS_1, but it's not compatible with Python 3.8, so it needs to be set individually in py36 and py37 builds.
+    # It is not being added, because the conda solver takes forever to resolve when pymapd is present, so it's not being tested for now
+    BACKENDS_1: "mysql parquet postgres sqlite"
```

Comment:
is there someway you can actually put the names rather than backends_1, e.g.
mysql-parquet-postgres-sqlite (e.g. more informative in the displays / code).